index.rst

```diff
@@ -1,52 +1,163 @@
 ..
-  Technote content.
-
   See https://developer.lsst.io/docs/rst_styleguide.html
   for a guide to reStructuredText writing.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+:tocdepth: 1
 
-  Use the following syntax for sections:
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Sections
-  ========
+.. sectnum::
 
-  and
+.. Add content below. Do not include the document title.
 
-  Subsections
-  -----------
+.. note::
 
-  and
+   **This technote is not yet published.**
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+   Summary of state of the art PSF estimation tools and their suitability for the LSST alert pipeline.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+Overview
+========
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-     :target: http://target.link/url
+Understanding the point-spread function is important for several Alert Production tasks, including image differencing and photometry. The LSST point spread function is expected to vary in a complex way with chip position, telescope orientation and focus, and atmospheric conditions. The PSF must be fit independently for each image (in particular, the effect of atmospheric turbulence is expected to change even between the two snaps of a visit), and must be fit quickly enough to not form a bottleneck for Alert Production.
 
-     Caption text.
+The requirements for PSF quality depend on downstream operations, but are in general poorly quantified. In particular:
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+- the Alard & Lupton ([AL1998]_) image differencing algorithm requires that the FHWM of the PSF be known to within 10% (Reiss, priv. comm. 2017), but is otherwise self-contained
```

Comment:
Does this need to be added to the RST style guide? It doesn't mention a `:cite` command at the moment.